streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -24,7 +24,7 @@
 
 import java.util.Collections;
 import java.util.Set;
-
+@SuppressWarnings("unchecked")
```

Comment:
This is a meta comment: we should think hard on not introducing unnecessary unchecked warnings, because they indicate that we are losing on strong typing.

Reply:
Ack, during refactoring  I'll remove `unchecked` warnings.